src/features/instantiate/containers/PartySelection.tsx

```diff
@@ -80,21 +80,36 @@ export const PartySelection = () => {
   const appMetadata = useAppSelector((state) => state.applicationMetadata.applicationMetadata);
   const selectedParty = useAppSelector((state) => state.party.selectedParty);
 
+  const appPromptForPartyOverride = useAppSelector(
+    (state) => state.applicationMetadata.applicationMetadata?.promptForParty,
+  );
+  const autoRedirect = useAppSelector((state) => state.party.autoRedirect);
+
+  const { langAsString, lang } = useLanguage();
+
   const [filterString, setFilterString] = React.useState('');
   const [numberOfPartiesShown, setNumberOfPartiesShown] = React.useState(4);
   const [showSubUnits, setShowSubUnits] = React.useState(true);
   const [showDeleted, setShowDeleted] = React.useState(false);
 
-  React.useEffect(() => {
-    dispatch(PartyActions.getParties());
-  }, [dispatch]);
+  const [hasSelected, setHasSelected] = React.useState(false);
+  const navigate = useNavigate();
 
   const onSelectParty = (party: IParty) => {
-    dispatch(PartyActions.selectParty({ party, redirect: true }));
+    dispatch(PartyActions.selectPartyFulfilled({ party: null }));
+    setHasSelected(true);
+    dispatch(PartyActions.selectParty({ party }));
     // Clear any previous instantiation errors.
     dispatch(InstantiationActions.instantiateRejected({ error: null }));
   };
 
+  const { org, app } = window as Window as IAltinnWindow;
+  useEffect(() => {
+    if (selectedParty && hasSelected) {
+      navigate('/');
+    }
+  }, [selectedParty, hasSelected, navigate, org, app]);
```

Comment:
Not quite sure what `org` and `app` is doing here

Reply:
Nice catch, probably forgot to remove this after some experimentation